README.rst

```diff
@@ -103,19 +103,24 @@ be specified in this file (without the preceding dashes), for example::
     count =
     quiet-level = 3
 
-This is equivalent to running::
-
-    codespell --quiet-level 3 --count --skip "*.po,*.ts,./src/3rdParty,./src/Test"
```

Comment:
Did you intend to remove this?

Reply:
Yeah with the "these both" suggestion I think it's clearer why 👍